custom_components/octopus_energy/heat_pump/zone.py

```diff
@@ -208,7 +209,7 @@ async def async_set_preset_mode(self, preset_mode):
         )
       else:
         zone_mode = self.get_zone_mode()
-        await self._client.async_set_heat_pump_zone_mode(self._account_id, self._heat_pump_id, self._zone.configuration.code, zone_mode, self._attr_target_temperature)
+        await self._client.async_set_heat_pump_zone_mode(self._account_id, self._heat_pump_id, self._zone.configuration.code, zone_mode, None)
```

Comment:
question: has this been changed to None for a reason?

Reply:
Same for this, the temperature passed in is otherwise ignored/overwritten?